recipes/assimp/5.0.x/conanfile.py

```diff
@@ -21,11 +21,13 @@ class Assimp(ConanFile):
         "shared": [True, False],
         "double_precision": [True, False],
         "fPIC": [True, False],
+        "no_export": [True, False],
     }
     default_options = {
         "shared": False,
         "double_precision": False,
         "fPIC": True,
+        "no_export": False,
```

Comment:
all "export" like options (except no_export) must be deleted in `configure` method if `no_export=True`